Sources/PayPalMessages/PayPalMessageModal.swift

```diff
@@ -165,6 +165,7 @@ final class PayPalMessageModal: UIViewController, WKUIDelegate {
     // MARK: - Config Functions
 
     private func configViews() {
+        view.accessibilityLabel = "PayPal Learn More Modal"
```

Comment:
Does this get read out anywhere since it doesn't appear to be an accessibility element? If not is this safe to remove?